concourse/pipelines/linux-image-build.yaml

```diff
@@ -1531,11 +1547,11 @@ jobs:
 - name: publish-to-prod-centos-7
   plan:
   - get: guest-test-infra
+    passed: [ image-test-centos-7 ]
```

Comment:
you've moved the trigger and passing conditions to a different resource. i don't think we want to track guest-test-infra through the pipeline

Reply:
Yes, the image-test job doesn't need gcs resource, so I move here. I understand that we don't track guest-test-infra resource. but the pass condition has to be on a get task

Reply:
alternately, you could consider adding the gcs resource to the test job solely for tracking the release, even though technically the image is what's used. or if we want the pipeline to reflect reality in that way, we could create and track a 'gce image' resource